ietf/static/js/ietf.js

```diff
@@ -46,6 +46,49 @@ $(document)
             .tooltip();
     });
 
+// Help browser to wrap long link texts (esp. email addresses) more sensibly.
+$(document)
+    .ready(function () {
+        $("#content a")
+            .each(function () {
+                // get the text of the <a> element
+                var text = $(this)
+                    .text();
+                // insert some <wbr> at strategic places
+                var newtext = text.replace(/([@._])/g, "$1<wbr>");
+                if (newtext === text) {
+                    return;
+                }
+                // now replace only that text inside the element's HTML
+                var newhtml = $(this)
+                    .html()
+                    .replace(text, newtext);
+                $(this)
+                    .html(newhtml);
+            });
+
+        // $("#content table.tablesorter")
```

Comment:
Did you mean to bring this in?

Reply:
Yes. This really helps with wrapping on narrow screens.